sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/FileFormatWriterSuite.scala

```diff
@@ -32,6 +33,24 @@ class FileFormatWriterSuite extends QueryTest with SharedSQLContext {
     }
   }
 
+  test("SPARK-23271 empty dataframe when saved in parquet should write a metadata only file") {
```

Comment:
If this is specific to `parquet`, can we have this `ParquetFileFormatSuite` instead?

Reply:
@dongjoon-hyun Thank you. Let me check if we have similar issue for orc. If not, i will move it to ParquetFileFormatSuite.

Reply:
Thank you @dilipbiswal . 
I checked with ORC, too. Your patch work for ORC two.
In this suite, can you extend the test case for ORC too?

Reply:
Thank you very much @dongjoon-hyun. You are super quick :-). Yes, i will add the test case for ORC.

Reply:
Ur, `FileBasedDataSourceSuite` may be more suitable. It has a similar test case. You can add your test case in the similar manner.
https://github.com/apache/spark/blob/master/sql/core/src/test/scala/org/apache/spark/sql/FileBasedDataSourceSuite.scala#L59-L73

Reply:
@dongjoon-hyun Sure. Will take a look. Thanks !!